File: src/main/java/com/naminhyeok/fantazzk/room/JoinRoomRequest.java
package com.naminhyeok.fantazzk.room;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;

@Schema(description = "방 참가 요청")
record JoinRoomRequest(
    @Schema(description = "참가자 닉네임", example = "게스트", requiredMode = Schema.RequiredMode.REQUIRED)
    @NotBlank(message = "닉네임은 비어 있을 수 없습니다") String nickname
) {
}

File: room/web/src/main/kotlin/com/naminhyeok/fantazzk/room/web/dto/JoinRoomRequest.kt
package com.naminhyeok.fantazzk.room.web.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "방 참가 요청입니다.")
data class JoinRoomRequest(
    @field:Schema(description = "방에 참가할 팀장 닉네임입니다.", example = "참가자")
    val nickname: String,
)
